fixtures/tests/_fixtures/test_popen.py

```diff
@@ -114,6 +114,15 @@ def test_kill(self):
         proc = FakeProcess({}, {})
         self.assertIs(None, proc.kill())
 
+    def test_poll(self):
+        proc = FakeProcess({}, {})
+        self.assertIs(None, proc.poll())
```

Comment:
I didn't modify the README since it's pretty terse at the moment. I think making it more comprehensive should be treated separately.